numbers/src/test/java/org/dicio/numbers/lang/fr/FrenchFormatter.java

```diff
@@ -0,0 +1,236 @@
+package org.dicio.numbers.lang.fr;
+
+import java.time.LocalTime;
+
+import org.dicio.numbers.formatter.NumberFormatter;
+import org.dicio.numbers.util.MixedFraction;
+
+public class FrenchFormatter extends NumberFormatter {
+
```

Comment:
I think you should also translate the following arrays from `EnglishFormatter` into French, otherwise the code below won't work: NUMBER_NAMES, NUMBER_NAMES_SHORT_SCALE, NUMBER_NAMES_LONG_SCALE, ORDINAL_NAMES, ORDINAL_NAMES_SHORT_SCALE, ORDINAL_NAMES_LONG_SCALE.

Reply:
Yes, will work on that. Just one question though: what is the difference between NUMBER_NAMES in the Formatter and their copy in tokenizer.json?

Reply:
The tokenizer contains the code that allows parsing numbers. And each number may have different words. But when we do formatting, it works the other way around, and so there should be one unique mapping from number to word. The two configurations can definitely be merged in some way, but for the moment I decided to keep it simple.